specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json

```diff
@@ -2066,6 +2066,23 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Resources/operations": {
+      "get": {
+        "tags": [
+          "Features"
+        ],
+        "operationId": "Features_Operations",
```

Comment:
fix operationId to reflect this is on Microsoft.Resources

Reply:
"/providers/Microsoft.Resources/operations" is only supported by apiversion 2015-05-01, but there is no such a folder, do we still need this api in resources.json?